.github/workflows/test.yml

```
@@ -0,0 +1,67 @@
+name: Test of Run Command with CASL2 Sample Files
+
+on:
+  push:
+  pull_request:
+
+jobs:
+
+  test_run:
+
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [windows-latest, ubuntu-latest, macos-latest]
+
+    name: Run Tests with Firefox and Chrome
+
+    runs-on: ${{ matrix.os }}
+
+    steps:
+      - uses: actions/checkout@v3
+
+      # Python 3.9をインストールする
+      - name: Set up Python 3.9
+        uses: actions/setup-python@v4
+        with:
+          python-version: 3.9
+
+      # ubuntu, macosのときはFirefoxの最新版をインストールする
+      - name: Set up Firefox on Ubuntu or MacOS
+        uses: browser-actions/setup-firefox@latest
+        if: matrix.os != 'windows-latest'
+        with:
+          firefox-version: 'latest'
+
+      # windowsのときはFirefoxの84.0をインストールする
```

コメントに latest が対応していないことを追記しておいて、latestへの対応があれば統合出来るように残しておくと良さそうです

```suggestion
      # windowsのときはFirefoxの84.0をインストールする
      # windowsではfirefox-version: "latest"をサポートしていないことが報告されています
      # https://github.com/browser-actions/setup-firefox/issues/252
```